scene/theme/default_theme.cpp

```diff
@@ -956,6 +956,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 		hue_texture->set_width(800);
 		hue_texture->set_height(6);
 		hue_texture->set_gradient(hue_gradient);
+		hue_texture->update_now();
```

Comment:
Is this needed for anything?

Reply:
I'm not entirely sure how the current implementation is supposed to work. If the texture still changes at the end of the frame/on the next frame, will it even render correctly all the time? It makes sense to me to make sure that the image is ready before we finish setting up the theme.

Reply:
Deferred calls are done at the end of the frame, which happens before rendering.
It should be easy to test, but IMO there won't be problems.